inspirehep/records/indexer/cli.py

```diff
@@ -20,7 +20,7 @@
 from inspirehep.records.api import InspireRecord
 from inspirehep.records.indexer.tasks import batch_index
 
-logger = logging.getLogger()
+LOGGER = logging.getLogger()
```

Comment:
```__name__```

Reply:
I don't know ho it didn't get in previous commit.